src/sas/qtgui/Utilities/Preferences/PreferencesPanel.py

```diff
@@ -87,10 +98,16 @@ def close(self):
         super(PreferencesPanel, self).close()
 
     def addWidget(self, widget: QWidget, name: Optional[str] = None):
+        """Add a single widget to the panel"""
+        # Set the parent of the new widget to the parent of this window
+        widget.parent = self.parent
         self.stackedWidget.addWidget(widget)
+        # Set display name in the listWidget with the priority of
+        #  widget.name > name passed to method > "Unknown"
         name = widget.name if hasattr(widget, 'name') and widget.name else name
         name = "Unknown" if not name else name
```

Comment:
if `widget.name` doesn't exist and the argument name is default, you should check for None rather than `not name`.
```
name = 'Unknown' if name is None else name
```